src/targets/gpu/compile_hip.cpp

```diff
@@ -205,8 +212,41 @@ compile_hip_src(const std::vector<src_file>& srcs, std::string params, const std
     return {prog.get_code_obj()};
 }
 
+std::vector<std::vector<char>>
+compile_hip_src(const std::vector<src_file>& srcs, std::string params, const std::string& arch)
+{
+    std::vector<hiprtc_src_file> hsrcs{srcs.begin(), srcs.end()};
+    auto p      = dynamic_loader::path(&compile_hip_src_with_hiprtc);
+    auto driver = p.parent_path().parent_path() / "bin" / "migraphx-hiprtc-driver";
+
+    if(fs::exists(driver))
+    {
+        value v;
+        v["srcs"]   = to_value(hsrcs);
+        v["params"] = to_value(params);
```

Comment:
these params would not have workarounds

Reply:
Of course not, the workarounds are added in `compile_hip_src_with_hiprtc`.

Reply:
But i don't see it calling it `compile_hip_src_with_hiprtc`. it only calls it if it fails to run through `hiprtc-driver`.

Reply:
The hiprtc-driver calls `compile_hip_src_with_hiprtc`.